package.json

```diff
@@ -21,6 +21,7 @@
         "@mui/icons-material": "^5.5.1",
         "@mui/lab": "^5.0.0-alpha.75",
         "@mui/material": "^5.5.3",
+        "@mui/x-data-grid": "^6.19.3",
```

Comment:
x-data-grid is not used in the gridsuite fronts. We don't want to have tens of tables solutions and libraries. 
You should one of the solution that are already used :
- aggrid
- VirtualizedTable
- MUI Table

Reply:
done, migrated to ag-grid